### DISCUSSION TITLE
Bed PID drives print head into the bed

### DISCUSSION THREAD

**Discussion by Dakweed666** [mriscoc/Ender3V2S1#1027] [General]
Updated my Ender 3 V2 NEO to the latest TJC screen firmware on the separate page, did a nozzle heating PID which worked all good, went to do a bed PID, as soon as I initiated it, the printhead moved to the front left corner of the bed and then just went downwards till it drove into the bed, the CR Touch probe was outside the edge of the bed, shut it off with the power switch and to scared to try it again, any ideas?

**Comment by mriscoc** [comment]
For the V2 Neo, Z home should be done at the center of the bed. Check if you flashed the BLT version. The MM (Manual mesh) version doesn't use CRTouch and do the Z homing at the front-left corner.

**Reply by Dakweed666** [reply]
Hi, tried it again today and it worked fine, strange, the time I tried it yesterday I didn't do an auto home, today, I rebooted, did an auto home then did the bed heater PID and it worked perfectly, strange.

**Reply by mriscoc** [reply]
Maybe you have a loose connection.
